CppCoreGuidelines.md

```diff
@@ -19031,6 +19032,30 @@ Failing to follow this results in difficult to diagnose errors for clients of a
 
 A test should verify that the header file itself compiles or that a cpp file which only includes the header file compiles.
 
+### <a name="Rs-incscope"></a>SF.12: Use the quoted or angle bracket form of `#include` to distinguish standard headers from others
+
+##### Reason
+
+To understand dependencies it is important to know where included files come from. Given the different search algorithms, 
+the angle form (`<>`) identifies standard headers, the quoted (`""`) everything else.
+
+##### Example
+
+    #include <string>       // From the standard library, use the <> form
+    #include "helpers.h"    // A project specific file, use "" form
+
+##### Note
+
+This is specified [here](http://eel.is/c++draft/cpp.include) in the standard.
```

Comment:
Seems like http://eel.is/c++draft/cpp.include#8 would be a slight improvement on the link